build_miniconda.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
why do you not like `-x` it would complete remove all the comments you put in your script.

They basically translate from "bash" to "english"

Reply:
I found the output of `-x` to be too verbose in that case and preferred to put `echo` statements in the script. I'll mark this as solved but if it's something really important for you we can set `-x`.